packages/@uppy/core/src/BasePlugin.js

```diff
@@ -42,6 +45,13 @@ module.exports = class BasePlugin {
     this.setPluginState() // so that UI re-renders with new options
   }
 
+  i18nInit () {
+    this.translator = new Translator([this.defaultLocale, this.uppy.locale, this.opts.locale])
+    this.i18n = this.translator.translate.bind(this.translator)
+    this.i18nArray = this.translator.translateArray.bind(this.translator)
```

Comment:
Could we move `translator` and `i18nArray` to private properties?

Reply:
Yes, you mean the new `#` thing? I think so, shouldn’t be public.

Reply:
i18nArray should be accessible by subclasses so they can use it in preact code